tmexp/__main__.py

```diff
@@ -209,16 +203,23 @@ def get_parser() -> argparse.ArgumentParser:
     train_hdp_parser.add_argument(
         "-i",
         "--input-dir",
-        help="First level input directory for the BoW files.",
-        required=True,
+        help="First level input directory for the BoW files, defaults to BOW_DIR.",
+        default=None,
         type=str,
     )
     train_hdp_parser.add_argument(
         "-o",
         "--output-dir",
         help="First level output directory for the word-topic and document-topic"
-        " distributions.",
-        required=True,
+        " distributions, defaults to TOPIC_DIR.",
+        default=None,
+        type=str,
+    )
+    train_hdp_parser.add_argument(
+        "--exp-name",
+        help="Name of the experiment, defaults to 'experiment_i' where i is the number "
+        "of other experiments for this dataset.",
+        default=None,
```

Comment:
We can use the date (MM-DD-HH:MM) instead of a number as default (or as default prefix, which I like more): more informative, sorts with string sort, etc.

Reply:
done